### PR TITLE
react-1269-skill-map-result-add-comment-section

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Skill Map Results - add comments section
[#1269](https://github.com/nerubia/kh360-react/issues/1269)

### CODE REVIEW

File: src/features/admin/skill-map-results/skill-map-results-list-table.tsx [hunk 1]
```diff
@@ -65,6 +65,20 @@ export const SkillMapResultsListTable = () => {
   useEffect(() => {
     if (user_skill_map.length > 0) {
       processData(user_skill_map)
+      const filteredDetail = user_skill_map
+        .filter((s) => s.skill_map_results.length > 0 && s.skill_map_results[0].comments !== "")
```
Comment (jatnerubia) [inline src/features/admin/skill-map-results/skill-map-results-list-table.tsx:69]: Move this to the sort util

**Review by jatnerubia — CHANGES_REQUESTED** [review]
Just a minor change ^^

File: src/features/admin/skill-map-results/skill-map-results-list-table.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/features/admin/skill-map-results/skill-map-results-list-table.tsx]: done :3

**Review by gborgonia-ir — APPROVED** [review]

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**